docs/configuration.txt

```diff
@@ -534,6 +549,43 @@ Blobs
   By default blobs will be stored under the same path as normal data.
   A relative path value is interpreted as relative to ``CRATE_HOME``.
 
+.. _ref-configuration-repositories:
+
+Repositories
+------------
```

Comment:
it is not clear that this section is for URL type, also the above section with path.repo - it is not clear that this is for fs type. so probably it would make sense to structure this document diferently e.g. by repo type